iotdb-core/ainode/iotdb/ainode/core/inference/pipeline/__init__.py

```diff
@@ -0,0 +1,29 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+from iotdb.ainode.core.inference.pipeline.sundial_pipeline import SundialPipeline
+from iotdb.ainode.core.inference.pipeline.timerxl_pipeline import TimerxlPipeline
+
+
+def get_pipeline(model_id, device):
+    if model_id == "timerxl":
+        return TimerxlPipeline(model_id, device=device)
+    elif model_id == "sundial":
+        return SundialPipeline(model_id, device=device)
```

Comment:
Under current implementation, to integrate a new model, we should add another if-else branch here? Is it toooooo much?